pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-indicator/default.nix

```diff
@@ -0,0 +1,33 @@
+{ autoreconfHook
+, fetchFromGitHub
+, glib
+, intltool
+, lib
+, libappindicator-gtk2
+, libtool
+, pidgin
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pidgin-indicator";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "philipl";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-CdA/aUu+CmCRbVBKpJGydicqFQa/rEsLWS3MBKlH2/M=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ glib intltool libappindicator-gtk2 libtool pidgin ];
+
+  meta = {
+    description = "A AppIndicator/KStatusNotifierItem Plugin for Pidgin";
```

Comment:
```suggestion
    description = "An AppIndicator/KStatusNotifierItem Plugin for Pidgin";
```

Reply:
I mean its not clear which should be used. An does not work with KStatusNotifierItem. Seems like either choice is valid to me.